Xamarin.Forms.Platform.iOS/Renderers/PageRenderer.cs

```diff
@@ -24,7 +24,8 @@ public class PageRenderer : UIViewController, IVisualElementRenderer, IEffectCon
 		Page Page => Element as Page;
 		IAccessibilityElementsController AccessibilityElementsController => this;
 		Thickness SafeAreaInsets => Page.On<PlatformConfiguration.iOS>().SafeAreaInsets();
-		bool IsPartOfShell => (Element?.Parent is BaseShellItem);
+		bool IsPartOfShell => (Element?.Parent is BaseShellItem || (
+			Element?.Parent is TabbedPage && Element?.Parent?.Parent is BaseShellItem));
```

Comment:
This doesn't seem to be used?

Reply:
The property already existed and is used (e.g. line 390, 404, 427), I just added another condition